tests/test_histogramming.py

```diff
@@ -0,0 +1,15 @@
+from plothist import make_hist
+import boost_histogram as bh
+
+
+def test_make_hist():
```

Comment:
Is this test only of simple example or an actual test we want to keep?
Should we add more tests in the PR?
Should we add something like `make_examples.py` in this PR?

Reply:
* We can keep this simple test. Even if it is trivial, it is important that it works.
* More complex tests should be created in dedicated PRs, since some may take longer than others to work.
* The testing of the figure can take time to implement, I would not block the implementation of other tests because of that.